### PR TITLE
Eugen/cherry pick superfluid test scaffolding updates

### PR DESCRIPTION
**Pull request by antstalepresh**
Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

File: x/superfluid/keeper/stake_test.go [hunk 1]
```diff
@@ -47,13 +46,14 @@ func (suite *KeeperTestSuite) SetupSuperfluidDelegations(delAddrs []sdk.AccAddre
 	for _, del := range superDelegations {
 		delAddr := delAddrs[del.delIndex]
 		valAddr := valAddrs[del.valIndex]
-		lock := suite.SetupSuperfluidDelegate(delAddr, valAddr, del.lpDenom, del.lpAmount)
+		lock := suite.SetupSuperfluidDelegate(delAddr, valAddr, denoms[del.lpIndex], del.lpAmount)
 		expAcc := types.NewSuperfluidIntermediaryAccount(lock.Coins[0].Denom, valAddr.String(), 0)
+		gotAcc := suite.App.SuperfluidKeeper.GetIntermediaryAccount(suite.Ctx, expAcc.GetAccAddress())
```
Comment (mattverse) [inline x/superfluid/keeper/stake_test.go:51]:
```suggestion
		address := suite.App.SuperfluidKeeper.GetLockIdIntermediaryAccountConnection(suite.Ctx, lock.ID)
		gotAcc := suite.App.SuperfluidKeeper.GetIntermediaryAccount(suite.Ctx, address)
```

What do you think about changing these lines as above? expAcc and gotAcc is confusing and hacky imo

**Review by mattverse — APPROVED** [review]
Small change for code sanity requested, otherwise LGTM!

File: x/superfluid/keeper/stake_test.go (hunk 1 above)
Comment (antstalepresh) [reply x/superfluid/keeper/stake_test.go:51]: Good suggestion

**Comment by antstalepresh** [comment]
> Small change for code sanity requested, otherwise LGTM!

Your suggestion is merged and checked the test pass

**Pull request merged**